src/styles/index.scss

```diff
@@ -1,3 +1,140 @@
+$main-accent-color: #060b35;
+$secondary-accent-color: #616070;
+$blue-accent-color: #00acdc;
+$white-accent-color: #fff;
+$font-family: Roboto, sans-serif;
+
+@import '../utils/veriables';
+@import '../styles/blocks/card';
+@import '../styles/blocks/stars';
+
 body {
   margin: 0;
 }
+
+.card {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__product-name {
+    margin-top: 40px;
+    margin-bottom: 0;
+    font-family: $font-family;
+    color: $main-accent-color;
+    width: 100%;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+  }
+
+  &__product-id {
+    margin-top: 4px;
+    margin-bottom: 0;
+    font-family: $font-family;
+    color: $secondary-accent-color;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+  }
+
+  &__price {
+    display: flex;
+    margin-top: 24px;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__price-value {
+    margin: 0;
+    font-family: $font-family;
+    font-size: 12px;
```

Comment:
The `font-weight` property should not have a unit. Change `font-weight: 400px;` to `font-weight: 400;`.